Formula/haxe.rb

```diff
@@ -39,8 +39,13 @@ class Haxe < Formula
 
   uses_from_macos "m4" => :build
   uses_from_macos "perl" => :build
+  uses_from_macos "rsync" => :build
   uses_from_macos "unzip" => :build
 
+  on_linux do
+    depends_on "node" => :test
```

Comment:
Alternatively, can disable JavaScript test or change to another language. On macOS, `osascript` is pre-installed and can run JavaScript.

Reply:
Since this is only a test dependency, I think it's fine.